### PR TITLE
templates: fetch fixed size of members' avatar

### PR DESCRIPTION
**Pull request by tc608**

### CODE REVIEW

File: templates/org/member/members.tmpl [hunk 1]
```diff
@@ -14,7 +14,7 @@
 			{{range .Members}}
 				<div class="item ui grid">
 					<div class="ui one wide column">
-						<img class="ui avatar" src="{{.RelAvatarLink}}?s=48">
+						<img class="ui avatar" src="{{.RelAvatarLink}}&s=48">
```
Comment (unknwon) [inline templates/org/member/members.tmpl]:
```suggestion
						<img class="ui avatar" src="{{AppendAvatarSize .RelAvatarLink 48}}">
```

**Comment by unknwon** [comment]
Thanks! Please see my suggestion.

**Comment by benyaminl** [comment]
@unknwon Will this be merged?

**Comment by unknwon** [comment]
@benyaminl Which version are you wanting this change to get in? We don't release patches just for minor improvements unless requested.

**Comment by benyaminl** [comment]
@unknwon It will be nice for next stable release included. Meanwhile I will patch via templates. Thank you 👍 🙏

**Comment by unknwon** [comment]
@benyaminl OK, will be included in 0.12.6, which is planned this weekend (not a commitment).

**Comment by unknwon** [comment]
The 0.12.6 has been released that includes the patch of the reported issue.

**Pull request merged**